src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    public LocalDate getLocalDate(String date) {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+        return LocalDate.parse(date, formatter);
+    }
+
+    public boolean isValidDate(LocalDate startDate, LocalDate endDate, String currentDate) {
```

Comment:
Please pay attention to the access modifiers, this method seems to be responsible for internal work of SalaryInfo, so it should be private